### PR TITLE
Create page with list of animals

### PR DESCRIPTION
**Pull request by alex-boklag**
- Created AnimalsPage component 
- Created AnimalsList component
- Created AnimalCard component
- Created helper for calculating animal's age
- Created Header component
- Created Footer component
- Set a structure of the project
image

### CODE REVIEW

File: right-for-life_front/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from "react";
```
Comment (AlexeyKasaev3) [inline right-for-life_front/src/App.jsx]: Можно Fragment не импортировать и вместо <Fragment> использовать <></>

File: right-for-life_front/src/components/AnimalCard/AnimalCard.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <button className="w-5/12 bg-green-300 hover:bg-green-400 text-green-700 font-bold py-2 px-2 rounded-xl mt-4 mb-2" style={{ outline: 'none'}}>
+          Приютить
+        </button>
+        <button className="w-5/12 bg-orange-300 hover:bg-orange-400 text-orange-700 font-bold py-2 px-2 rounded-xl mb-4 ml-2" style={{ outline: 'none'}}>
+          История
+        </button>
+      </div>
+    </li >
+  );
+}
```
Comment (AlexeyKasaev3) [inline right-for-life_front/src/components/AnimalCard/AnimalCard.jsx]: Размер табуляции то 2 пробела, то 4 пробела

File: right-for-life_front/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import logo from "../../assets/logo.svg";
```
Comment (AlexeyKasaev3) [inline right-for-life_front/src/components/Footer/Footer.jsx]: SVG лучше импортировать как `import { ReactComponent as Logo } from './logo.svg';`  и использовать потом как компонент `<Logo />`

File: right-for-life_front/src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import logo from "../../assets/logo.svg";
+
+export const Header = () => {
+	return (
+		<ul className="fixed w-full h-16 bg-lightgray-100 shadow-md flex justify-center items-center font-bold text-2xl text-gray-600 z-50">
```
Comment (AlexeyKasaev3) [inline right-for-life_front/src/components/Header/Header.jsx]: Для соблюдения семантики лучше обернуть все в `<header>` и `<nav>`

File: right-for-life_front/src/App.jsx (hunk 1 above)
Comment (alex-boklag) [reply right-for-life_front/src/App.jsx]: Это просто сокращённая запись, не вижу большой разницы

File: right-for-life_front/src/components/AnimalsList/AnimalsList.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { AnimalCard } from "../../components/AnimalCard/AnimalCard";
+
+export const AnimalsList = ({ animals }) => {
+	return (
+		<ul className="flex flex-wrap justify-center">
+			{animals.map(animal => {
+				return <AnimalCard key={animal.id.toString()} animal={animal} />;
```
Comment (AlexeyKasaev3) [inline right-for-life_front/src/components/AnimalsList/AnimalsList.jsx]: А зачем в `<AnimalCard key` делать  `.toString()` ?

Блин, все же круто быть код ревьюером. Ничего не делаешь, только умничаешь )))

File: right-for-life_front/src/components/AnimalCard/AnimalCard.jsx (hunk 2 above)
Comment (alex-boklag) [reply right-for-life_front/src/components/AnimalCard/AnimalCard.jsx]: Везде 2 пробела

Comment (AlexeyKasaev3) [reply right-for-life_front/src/components/AnimalCard/AnimalCard.jsx]: AnimalsList.jsx 4 пробела

File: right-for-life_front/src/components/AnimalCard/AnimalCard.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import calcAge from '../../heplers/calcAge';
```
Comment (yevhverb) [inline right-for-life_front/src/components/AnimalCard/AnimalCard.jsx]: remane folder "he**pl**ers" to "he**lp**ers"

File: right-for-life_front/src/components/Header/Header.jsx (hunk 4 above)
Comment (yevhverb) [inline right-for-life_front/src/components/Header/Header.jsx]: don't use fixed height, use padding top and bottom

Comment (yevhverb) [inline right-for-life_front/src/components/Header/Header.jsx]: don't use fixed height, use padding top and bottom

File: right-for-life_front/src/components/Footer/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+export const Footer = () => {
+	return (
+		<ul className="h-16 bg-lightgray-100 shadow-md flex justify-center items-center font-bold text-2xl text-gray-600">
```
Comment (yevhverb) [inline right-for-life_front/src/components/Footer/Footer.jsx]: don't use fixed height, use padding top and bottom

File: right-for-life_front/src/containers/AnimalsPage/AnimalsPage.jsx [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+	const [animals, setAnimals] = useState([]);
+
+	useEffect(() => {
+		fetch(dataApi)
+			.then(data => data.json())
+			.then(list => setAnimals(list));
+	}, []);
+
+	return (
+		<div className="animals-page bg-lightgray-300 pt-20 pb-4">
```
Comment (yevhverb) [inline right-for-life_front/src/containers/AnimalsPage/AnimalsPage.jsx]: make bg color "bg-lightgray-100"

File: right-for-life_front/src/components/AnimalCard/AnimalCard.jsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+export const AnimalCard = (props) => {
+  const { name, age, photos } = props.animal;
+
+  return (
+    <li className="w-64 m-4 flex flex-col">
+      <div className="z-40 pointer">
+        <div className="h-56 rounded-xl bg-cover bg-top" style={{ backgroundImage: "url(" + photos[0] + ")" }}></div>
+      </div>
+      <div className="w-11/12 bg-gray-100 text-gray-700 font-bold text-center shadow-xl rounded-b-xl self-center">
```
Comment (yevhverb) [inline right-for-life_front/src/components/AnimalCard/AnimalCard.jsx]: make bg color for info "bg-white"

**Comment by alex-boklag** [comment]
> Maybe I do not understand the purpose of this request, but the component is raw. To finish this task, I think you need to add pagination and fix age formatting: make it more accurate and add declination (1 год, 2 года, 5 лет). Currently, if age is less than 30 days, it shows 1 month and does not care that maybe a month not contains 30 days

Yeah, you don't understand the purpose of this pull request. 
1. It's the first version of the component. I think that pagination and correct age can be added later.
2. In this pull request, I have defined a common structure of the project.
3. This pull request will allow config react-router and redux.

**Comment by maksv21** [comment]
> Yeah, you don't understand the purpose of this pull request.
> 
> 1. It's the first version of the component. I think that pagination and correct age can be added later.
> 2. In this pull request, I have defined a common structure of the project.
> 3. This pull request will allow config react-router and redux.

I think it makes no sense to make several versions of a component if you can finish it now. But if it's for architecture, then okay

**Comment by alex-boklag** [comment]
Close #2

**Pull request merged**